onnxruntime/test/onnx/TestCase.cc

```diff
@@ -203,7 +204,14 @@ class OnnxModelInfo : public TestModelInfo {
     if (!model_pb.ParseFromZeroCopyStream(&f)) {
       ORT_THROW("Failed to load model because protobuf parsing failed.");
     }
-
+#ifdef __GNUG__
+    std::string url_string{model_url};
+#else
+    std::wstring url_string{model_url};
+#endif
+    if (url_string.size() >= 51) {
+      onnx_commit_ = {url_string.begin()+11, url_string.begin()+51};  
```

Comment:
>11 [](start = 41, length = 2)

SO this means that if the CI build config changes, and the amount of chars changes before the commit, then this code is broken?

Reply:
Yes - we could of course swtich to regex, but still not pretty much reliable though.